### PR TITLE
Update dependencies and remove tslint leftovers

### PR DESCRIPTION
**Pull request by maciaszczykm**

### CODE REVIEW

**Comment by maciaszczykm** [comment]
/assign @shu-mutou 

Can you review it?

File: docs/developer/code-conventions.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ We use following tools and settings for each check and formatting:
 | code    | tools                                                                    | setting |
 |---------|--------------------------------------------------------------------------|---------|
 | go      | [golangci-lint](https://github.com/golangci/golangci-lint)               | [`.golangci.yml`](../../.golangci.yml) |
-| ts      | [gts](https://github.com/google/gts)                                     | [`tslint.json`](../../tslint.json) |
+| ts      | [gts](https://github.com/google/gts)                                     | [`package.json`](../../package.json) |
```
Comment (shu-mutou) [inline docs/developer/code-conventions.md:41]: We need to maintain `.prettierrc.js` also.

**Comment by shu-mutou** [comment]
I think the others are good and works well.

File: docs/developer/code-conventions.md (hunk 1 above)
Comment (shu-mutou) [reply docs/developer/code-conventions.md:41]: So, following would be better
```
`eslintConfig` in [`package.json`](../../package.json) and [`.prettierrc.js`](../../.prettierrc.js)
```

**Comment by maciaszczykm** [comment]
@shu-mutou Thanks, PTAL.

**Comment by shu-mutou** [comment]
/lgtm
Thanks!

**Pull request merged**